### PR TITLE
Phase plot changes

### PR DESCRIPTION
**Pull request by iantaylor-NOAA**
This branch includes changes to the phase plot discussed in #703. 
I don't think these need a formal review but I'll let the github actions complete the testing before merging.

### CODE REVIEW

File: R/SSplotSPR.R [hunk 1]
```diff
@@ -28,6 +28,9 @@
 #' sprtarg input here will cause no horizontal line to be plotted.
 #' @param btarg target depletion to be used in plots showing depletion. May be
 #' omitted by setting to NA. "default" chooses based on model output.
+#' @param minbthresh minimum biomass thresholdto be used in plots
```
Comment (k-doering-NOAA) [inline R/SSplotSPR.R:31]: missing space between "threshold to"

File: R/SSplotSPR.R [hunk 2]
```diff
@@ -45,7 +48,7 @@ SSplotSPR <-
            uncertainty = TRUE,
            subplots = 1:4, forecastplot = FALSE,
            col1 = "black", col2 = "blue", col3 = "green3", col4 = "red",
-           sprtarg = "default", btarg = "default",
+           sprtarg = "default", btarg = "default", minbthresh = "default",
```
Comment (k-doering-NOAA) [inline R/SSplotSPR.R:51]: I wonder if this should instead be `minbthresh = replist[["minbthresh"]]`, as I think making defaults directly assigned when they are simple enough is supposed to be a good practice? Then there is no need to explain what default means in the documentation.

File: R/SSplotSPR.R [hunk 3]
```diff
@@ -427,28 +446,59 @@ SSplotSPR <-
...
+      # if SPRratio denominator is not SPR target,
+      # then add a line at sprtarg
+      if (sprtarg > 0 & SPRratioLabel == "1-SPR") {
+        abline(
+          h = 1 - sprtarg,
           lty = 2,
           col = rgb(0, 0, 0, 0.4)
         )
+        # caption <- paste0(
+        #   caption,
```
[the remark this answers is not in the document]
Comment (iantaylor-NOAA) [reply R/SSplotSPR.R:468]: Looks like it was unfinished changes. Needed revision and moving around.

File: R/SSplotSPR.R (hunk 2 above)
Comment (iantaylor-NOAA) [reply R/SSplotSPR.R:51]: I will open a separate issue to apply this to other functions.

File: R/SSplotSPR.R (hunk 3 above)
[the remark this answers is not in the document]
Comment (iantaylor-NOAA) [reply R/SSplotSPR.R:468]: Done.

**Pull request merged**